context/app/static/js/components/savedLists/CreateListDialog/CreateListDialog.jsx

```diff
@@ -25,14 +28,24 @@ function CreateListDialog({ dialogIsOpen, setDialogIsOpen }) {
     setDialogIsOpen(false);
   };
 
+  function handleExit() {
+    setTitle('');
+    setDescription('');
+  }
+
   function handleSubmit() {
-    createList({ title, description });
-    setDialogIsOpen(false);
+    if (!(title in savedLists)) {
+      createList({ title, description });
+      setDialogIsOpen(false);
+    } else {
+      setShouldDisplayWarning(true);
+    }
   }
 
   return (
     <DialogModal
       title="Create New List"
+      warning={shouldDisplayWarning && 'A list with that title already exists.'}
```

Comment:
```suggestion
      warning={shouldDisplayDuplicateTitleWarning && 'A list with that title already exists.'}
```
... In case there are other failure modes in the future? ... but it might be getting too long?

Reply:
I'll modify in the future if we need another warning. I'll also add this to the list of feedback areas.